src/rebar3_mini_typer.erl

```diff
@@ -272,7 +272,7 @@ write_inc_files(Inc, Analysis) ->
              msg(debug, "Records ~tp", [Info#info.records], Analysis),
              write_typed_file(File, Info, Analysis)
           end,
-    lists:foreach(Fun, dict:fetch_keys(Inc#inc.map)).
```

Comment:
My idea was to actually remove the middle-man/abstraction.
Stop using `map_dict*` functions and directly use maps wherever possible.
As you can see on this line, for instance, the abstraction was broken anyway (They used a `dict` function on something that they shouldn't had known to be a map).